.github/workflows/java.yml

```diff
@@ -60,5 +60,29 @@ jobs:
   call-publish-to-maven-central-workflow:
```

Comment:
Rename this job to `publish-to-maven-central` since it's no longer calling another workflow.